mesheryctl/pkg/utils/helpers.go

```diff
@@ -446,6 +447,65 @@ func StringInSlice(str string, slice []string) bool {
 	return false
 }
 
+// GetID returns a array of IDs from meshery server endpoint /api/{configurations}
+func GetID(configuration string) ([]string, error) {
+	url := MesheryEndpoint + "/api/" + configuration + "?page_size=10000"
+	configType := configuration + "s"
```

Comment:
Although I have no major issues with it, hardcoding an `s` at the end seems rather hacky, would it be feasible to change something in the payload we get?

If you think what I suggested is unnecessary you can ignore it.
If you think that it makes sense but isn't urgent, you can add a TODO comment and create an issue ticket at an appropriate time
Else you can make this change in this PR

Reply:
+1 for this, although it's a minor one, we might need to have a look

Reply:
@hershd23 @alphaX86 The reason to use ```s``` was because of the response we are getting.